### DISCUSSION TITLE
Translate dynamic nested routes

### DISCUSSION THREAD

**Discussion by oscarromerosilva** [amannn/next-intl#842] [Q&A]
It's possible to create this kind of routes with translations?

My idea is create a folder structure like this:
 ![image](https://github.com/amannn/next-intl/assets/37908558/fc46ccce-6a58-4350-a630-55f5acf72e39)
 
 whit this i'm trying to my route looks like:
`www.domain.com/spain/es/usuarios`,
`www.domain.com/italy/it/utenti`
`www.domain.com/germany/de/benutzer`

**Comment by amannn** [comment]
This is discussed in detail here: https://github.com/amannn/next-intl/issues/653
